package.json

```diff
@@ -37,9 +37,11 @@
   },
   "dependencies": {
     "amd-name-resolver": "0.0.6",
+    "babel-plugin-transform-es2015-modules-amd": "^6.22.0",
+    "babel-preset-env": "^1.2.0",
     "bower-config": "^1.3.0",
     "bower-endpoint-parser": "0.2.2",
-    "broccoli-babel-transpiler": "^5.6.2",
+    "broccoli-babel-transpiler": "6.0.0-alpha.3",
```

Comment:
This is the primary "merge blocker".  Need to release as something > .alpha to land (IMO).